lib/misc/cells.coffee

```diff
@@ -0,0 +1,32 @@
+scopes = require './scopes'
+{values} = require 'underscore-plus'
+
+module.exports =
+
+  getCurrentCell: (ed) ->
+    buffer = ed.getBuffer()
+    start = buffer.getFirstPosition()
+    end = buffer.getEndPosition()
+    regexString = atom.config.get 'julia-client.cellDelimiter'
+
+    unless regexString?
```

Comment:
`regexString` should never be `null` or `undefined`, no? `atom.config.get` could return an empty string, but that wouldn't be caught by this (and shouldn't matter anyways).